packages/synapse-react-client/src/components/styled/LightTooltip.tsx

```diff
@@ -18,7 +24,10 @@ export const LightTooltip: StyledComponent<TooltipProps> = styled(
       boxShadow: theme.shadows[1],
       border: `1px solid ${theme.palette.grey[500]}`,
     },
-    color: theme.palette.common.white,
+    color: theme.palette.background.paper,
+  },
+  [`& .${linkClasses.root}`]: {
+    color: theme.palette.primary.main,
   },
```

Comment:
In our `DefaultTheme`, links in tooltips are hard-coded to white text, because our normal tooltips have a dark background.

For this case, we have a light background, we need to go back to using the default link color, which is `primary.main`.